e2e/content-services/metadata/aspect-oriented-config.e2e.ts

```diff
@@ -122,8 +122,6 @@ describe('Aspect oriented config', () => {
         await viewerPage.clickInfoButton();
         await viewerPage.checkInfoSideBarIsDisplayed();
         await metadataViewPage.clickOnPropertiesTab();
-        await metadataViewPage.informationButtonIsDisplayed();
```

Comment:
looks like informationButtonIsDisplayed function is no more used in ADF. Can we remove that function completely?